.github/stale.yml

```
@@ -11,15 +11,18 @@ exemptLabels:
 # Label to use when marking an issue as stale
 staleLabel: stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
-markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs.
+markComment: |
+  This issue has been marked as *stale* because it did not see any
```

I'd use present perfect

```suggestion
  This issue has been marked as *stale* because it hasn't seen any
```